Cinema system/Cinema system/Controllers/CinemasController.cs

```diff
@@ -14,254 +12,171 @@ public class CinemasController : ControllerBase
 
         private readonly ICinemaService _сinemaService;
 
+        private const string ErrorOfCinemaNonexistence = "This cinema does not exists.";
+        private const string ErrorOfCinemaExistence = "This cinema is already exists.";
+        private const string ErrorOfHallNonexistence = "This hall does not exist in this cinema.";
+        private const string ErrorOfHallExistence = "This hall is already exists in this cinema.";
+
         public CinemasController(ICinemaService сinemaService)
         {
             _сinemaService = сinemaService;
         }
 
-        private IEnumerable<string> ErrorMessages(ModelStateDictionary model)
-        {
-            return model.Values
-                .SelectMany(x => x.Errors)
-                .Select(x => x.ErrorMessage);
-        }
-
         [HttpGet]
         public IActionResult GetCinemas()
         {
-            var response = new GetResponse<IEnumerable<CinemaView>>
-            {
-                IsSuccessful = true,
-                RequestedData = _сinemaService.GetCinemas()
-            };
-
-            return Ok(response);
+            return Ok(new GetResponse<IEnumerable<CinemaView>>(_сinemaService.GetCinemas()));
```

Comment:
Here I would probably assign result of a service call to a separate variable just for better readability.
And type argument (i.e. <IEnumerable<>>) should be inferred by a compiler, so just new GetResponse() should be enough.

Reply:
It gives an error when I write :
new GetResponse(cinema)